onnx_ops/onnx/node/generated/test_scan_sum/run_module_io_flags.txt

```diff
@@ -1,4 +1,4 @@
---input=2xf32=@input_0.bin
---input=3x2xf32=@input_1.bin
---expected_output=2xf32=@output_0.bin
---expected_output=3x2xf32=@output_1.bin
+--input=1x2xf32=@input_0.bin
+--input=1x3x2xf32=@input_1.bin
+--expected_output=1x2xf32=@output_0.bin
+--expected_output=1x3x2xf32=@output_1.bin
```

Comment:
These (auto-generated) changes flipped the test from passing to failing. Weird...